### PR TITLE
Standardize Handlebars Parameters, files that got missed

### PR DESCRIPTION
**Pull request by christiemolloy**
This PR closes issue #798 

Some files were left out of [#757 PR](https://github.com/patternfly/patternfly-next/pull/757/files) before it was merged. This PR serves to add these files back in, plus add in any other files that needed to be cleaned up to align with the handlebars parameters.

### CODE REVIEW

File: src/patternfly/components/Progress/progress-indicator.hbs [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-      {{#unless progress--no-measure}}
-        {{>progress__measure}}
-      {{/unless}}
-    {{/if}}    
-  </div>
\ No newline at end of file
+<div class="pf-c-progress__indicator{{#if progress-indicator--modifier}} {{progress-indicator--modifier}}{{/if}}" 
+  style="width: {{progress__value}}%;"
+  {{#if progress-indicator--attribute}}
+      {{{progress-indicator--attribute}}}
```
Comment (matthewcarleton) [inline src/patternfly/components/Progress/progress-indicator.hbs]: -1 space :)

File: src/patternfly/layouts/Level/level-group.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="pf-l-level__group"{{#if level-group--modifier}} {{level-group--modifier}}{{/if}}
```
Comment (matthewcarleton) [inline src/patternfly/layouts/Level/level-group.hbs]: can you put the modifier inside the ""

File: src/patternfly/layouts/Split/examples/split-example.hbs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{{#> split }}
+{{#> split}}
   {{#> split-item }}
```
Comment (matthewcarleton) [inline src/patternfly/layouts/Split/examples/split-example.hbs]: missed a space here

Comment (matthewcarleton) [reply src/patternfly/layouts/Split/examples/split-example.hbs]: and on line 8

**Comment by matthewcarleton** [comment]
Looks great! just a few little tweaks

File: src/patternfly/components/Dropdown/dropdown-basic.hbs [hunk 4]
```diff
@@ -1,3 +1,2 @@
 <div class="pf-c-dropdown__menu" {{#unless dropdown--IsExpanded}}hidden{{/unless}}>
```
Comment (michael-coker) [inline src/patternfly/components/Dropdown/dropdown-basic.hbs]: should we add `--modifier` and `--attribute` and `{{> @partial-block}}` to this? Also it's worth noting that this menu is empty in the example code. Not sure if fixing that is in this scope of this PR or not.

File: src/patternfly/layouts/Bullseye/examples/bullseye-example.hbs [hunk 5]
```diff
@@ -1,3 +1,3 @@
-{{#> bullseye }}
-  {{#> bullseye-item}}content{{/ bullseye-item}}
+{{#> bullseye}}
+  {{#> bullseye-item}}content{{/bullseye-item}}
 {{/ bullseye}}
```
Comment (michael-coker) [inline src/patternfly/layouts/Bullseye/examples/bullseye-example.hbs]: can you remove the space after the `/`?

**Comment by michael-coker** [comment]
looks like there are still some trailing spaces before the closing curly braces.

```
src/patternfly//components/Dropdown/dropdown-multi-select.hbs:            {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-apples" name="fruit"') }}{{/check-input}}
src/patternfly//components/Dropdown/dropdown-multi-select.hbs:            {{#> check-label check-label--attribute=(concat 'for="' id '-apples"') }}Apples{{/check-label}}
src/patternfly//components/Form/examples/form-horizontal-align-labels-example.hbs:  {{#> form-group }}
src/patternfly//components/Form/form-helper-text.hbs:  {{> @partial-block }}
src/patternfly//components/Card/examples/card-no-header-example.hbs:{{#> card }}
src/patternfly//components/Title/title.hbs:  {{> @partial-block }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:  {{#> dropdown id="dropdown-example-collapsed1" dropdown--HasToggleIcon="true" }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs: {{#> input-group-text inputGroupTextType="label" input-group-text--HasCalendarIcon="true" input-group-text--attribute='for="textInput9"' }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:  {{#> button button--modifier="pf-m-tertiary" button--HasSearchIcon="true" button--attribute='aria-label="search button for search input"' }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:{{#> input-group }}
src/patternfly//components/InputGroup/examples/input-group-example.hbs:  {{#> button button--modifier="pf-m-tertiary" button--HasQuestionCircleIcon="true" button--attribute='aria-label="popover for input"' }}
src/patternfly//layouts/Split/examples/split-example.hbs:  {{#> split-item }}
src/patternfly//layouts/Split/examples/split-example.hbs:  {{#> split-item }}
src/patternfly//layouts/Split/examples/split-gutters-example.hbs:  {{#> split-item }}
src/patternfly//layouts/Split/examples/split-gutters-example.hbs:  {{#> split-item }}
src/patternfly//layouts/Bullseye/bullseye.hbs:  {{> @partial-block }}
src/patternfly//demos/BasicForms/examples/horizontal-form-demo-example.hbs:    {{#> form-group }}
src/patternfly//demos/BasicForms/examples/horizontal-form-demo-example.hbs:    {{#> form-group }}
src/patternfly//demos/BasicForms/examples/horizontal-form-demo-example.hbs:      {{#> form-label form-label--attribute='for="horizontal-form-email"' }}Email{{/form-label}}
src/patternfly//demos/BasicForms/examples/horizontal-form-demo-example.hbs:    {{#> form-group }}
src/patternfly//demos/BasicForms/examples/horizontal-form-demo-example.hbs:    {{#> form-group }}
src/patternfly//demos/BasicForms/examples/simple-form-demo-example.hbs:      {{#> form-label form-label--attribute='for="simple-form-number"' }}Phone Number{{/form-label}}
src/patternfly//demos/Toolbar/examples/toolbar-complex-example.hbs:        {{#> input-group }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:      {{#> card }}
src/patternfly//demos/PageLayout/page-template-gallery.hbs:        {{#> card-body }}
```

**Comment by michael-coker** [comment]
also should remove the space after the `/` in these files

```
src/patternfly//components/AboutModalBox/examples/about-modal-box-example.hbs:  {{/ about-modal-box-hero}}
src/patternfly//components/Content/examples/content-template-example.hbs:{{/ content}}
src/patternfly//layouts/Bullseye/examples/bullseye-example.hbs:{{/ bullseye}}
src/patternfly//layouts/Level/examples/level-three-example.hbs:  {{/ level-item}}
src/patternfly//layouts/Level/examples/level-three-example.hbs:  {{/ level-item}}
src/patternfly//layouts/Level/examples/level-three-example.hbs:  {{/ level-item}}
src/patternfly//layouts/Level/examples/level-three-example.hbs:{{/ level}}
src/patternfly//layouts/Level/examples/level-two-example.hbs:  {{/ level-item}}
src/patternfly//layouts/Level/examples/level-two-example.hbs:  {{/ level-item}}
src/patternfly//layouts/Level/examples/level-two-example.hbs:{{/ level}}
```

File: src/patternfly/components/Dropdown/dropdown-basic.hbs (hunk 4 above)
Comment (matthewcarleton) [reply src/patternfly/components/Dropdown/dropdown-basic.hbs]: I'd say yes for consistency sake.

File: src/patternfly/components/Progress/progress-indicator.hbs (hunk 1 above)
Comment (christiemolloy) [reply src/patternfly/components/Progress/progress-indicator.hbs]: fixed

File: src/patternfly/layouts/Level/level-group.hbs (hunk 2 above)
Comment (christiemolloy) [reply src/patternfly/layouts/Level/level-group.hbs]: yes, good find!

File: src/patternfly/layouts/Split/examples/split-example.hbs (hunk 3 above)
Comment (christiemolloy) [reply src/patternfly/layouts/Split/examples/split-example.hbs]: fixed

File: src/patternfly/components/Dropdown/dropdown-basic.hbs (hunk 4 above)
Comment (christiemolloy) [reply src/patternfly/components/Dropdown/dropdown-basic.hbs]: added modifier, attribute and partial-block. Looks like this file if for the dropdown-basic element, but it uses pf-c-dropdown__menu  as the class, which already has its own file. So should I remove this one?

File: src/patternfly/layouts/Bullseye/examples/bullseye-example.hbs (hunk 5 above)
Comment (christiemolloy) [reply src/patternfly/layouts/Bullseye/examples/bullseye-example.hbs]: yep

File: src/patternfly/layouts/Split/examples/split-example.hbs (hunk 3 above)
Comment (michael-coker) [reply src/patternfly/layouts/Split/examples/split-example.hbs]: still seeing a space at the end `{{#> split-item }}`

File: src/patternfly/components/Title/title.hbs [hunk 6]
```diff
@@ -1,6 +1,6 @@
-<{{ titleType }} class="pf-c-title{{#if title--modifier}} {{title--modifier}}{{/if}}"
+<{{titleType}} class="pf-c-title{{#if title--modifier}} {{title--modifier}}{{/if}}"
 	{{#if title--attribute}}
-   {{{ title--attribute}}}
+   {{{title--attribute}}}
  	{{/if}}>
   {{> @partial-block }}
```
Comment (michael-coker) [inline src/patternfly/components/Title/title.hbs]: space at the end here - `{{> @partial-block }}`

**Comment by michael-coker** [comment]
took a pass through and looks like there are some more places to clean up spaces and formatting. went ahead and submitted a PR instead of leaving a bunch of comments :) https://github.com/christiemolloy/patternfly-next/pull/2

**Comment by christiemolloy** [comment]
@michael-coker the changes you are suggesting, I had made on my local machine. I guess they are not showing here?

**Comment by christiemolloy** [comment]
@michael-coker I merged your PR into this one, I think it should be good to merge!

**Comment by michael-coker** [comment]
Christie, great work on this. Just one more thing I found - in `Form/form-label.hbs`, the `{{#if required}}` block needs to go before the `{{#if form-label--attribute}}` block (or just within the element tag)

```
<label class="pf-c-form__label{{#if form-label--modifier}} {{form-label--modifier}}{{/if}}" 
  {{#if form-label--attribute}}
    {{{form-label--attribute}}}
  {{/if}}>
	{{#if required}}
    required
  {{/if}}
```

**Pull request merged**